integreat_cms/cms/fixtures/test_data.json

```diff
@@ -1885,6 +1919,40 @@
       "page": 5
     }
   },
+  {
+    "model": "cms.pagetranslation",
+    "pk": 82,
+    "fields": {
+      "title": "Teste Hidden Language",
+      "slug": "teste-hidden-language",
+      "status": "PUBLIC",
+      "content": "<p>Dies ist ein Inhalt in \"Hidden language\"</p>",
+      "language": 7,
```

Comment:
Also it's not possible to create a page without German translation.
So it's probably better to have it in test data as well, for consistency

Reply:
Yes, I agree! :+1:

Reply:
@timoludwig side question: in this test we have english translation with title, but with empty content.
But when there is no english translation at all (no entry in DB), API request for "en" language doesn't return this page at all. Is it an expected behavior? Shouldn't it also return a page with the same fallback message?

Reply:
No, this is the expected behavior. For pages, the fallback texts are only shown then the object exists in the database, but with an empty title.
Reason for this is that some regions want to create content for a specific target group, e.g. they add an info for people who speak Romanian but since this info is irrelevant for all other people, they want to hide it for them.
So we decided that the fallback mechanism for pages only works when at least the title is translated.
Also, a related issue is #1441.